### DISCUSSION TITLE
Topic page

### DISCUSSION THREAD

**Discussion by mak7ilenin** [VirumaaAnts/-Morality#6] [Q&A] [answered]
Need some thoughts about design on topic page. What do you think?

**Comment by mak7ilenin** [comment]
As we have the different colors on each topic, so we can set the theme of this page by changing the bg color on topic color.

**Reply by 9I6JloKo** [reply]
U can still do nothing bcs Popova is going nakhyi)))

**Comment by 9I6JloKo** [comment, marked as answer]
As we have the different colors on each topic, so we can set the theme of this page by changing the bg color on topic color.
